_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:60
 msgid "### Ping times and latency start well, then get worse, causing issues"
-msgstr ""
+msgstr "### Ping-Zeiten und Latenzzeiten beginnen gut, werden dann schlechter und verursachen Probleme"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:62
 msgid "This can indicate something else is competing with Jamulus on your network, so make sure nobody is watching HD Netflix movies or taking part in Zoom video conferences while you are playing. A more permanent solution for technically-minded users may be found by investigating [the issue of buffer bloat](https://www.bufferbloat.net/projects/bloat/wiki/) on their router and whether you can implement Smart Queue Management (SQM). [More details here](https://www.bufferbloat.net/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/)."
-msgstr ""
+msgstr "Dies kann darauf hindeuten, dass etwas anderes mit Jamulus in deinem Netzwerk konkurriert. Stelle also sicher, dass niemand HD-Netflix-Filme ansieht oder an Zoom-Videokonferenzen teilnimmt, während du spielst. Eine dauerhaftere Lösung für technisch versierte Benutzer kann darin bestehen, [das Problem der Pufferüberlastung](https://www.bufferbloat.net/projects/bloat/wiki/) auf deinem Router zu untersuchen und zu prüfen, ob du Smart Queue Management (SQM) implementieren kannst. [Weitere Einzelheiten hier](https://www.bufferbloat.net/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/)."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:64
 msgid "### Getting frustrated with software channels, audio routing, sample rates and more?"
-msgstr ""
+msgstr "### Du bist frustriert von Software-Kanälen, Audio-Routing, Abtastraten und mehr?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:66
 msgid "It's usually far easier and more reliable to have a [mixing desk](https://www.thomann.de/pics/bdb/191244/7355025_800.jpg) to connect your kit (instruments, mic, recorder etc.) and then send a simple stereo signal to your sound interface (be sure to listen to the resulting sound from the Jamulus Server via your computer though!). The huge variety of possible hardware, software and instrument combinations means that setting up your sound card to work with your particular configuration can otherwise get complicated very fast."
-msgstr ""
+msgstr "In der Regel ist es viel einfacher und zuverlässiger, ein [Mischpult](https://www.thomann.de/pics/bdb/191244/7355025_800.jpg) zu haben, an das du dein Equipment (Instrumente, Mikrofon, Aufnahmegerät usw.) anschließt und dann ein einfaches Stereosignal an dein Sound-Interface zu senden (achte aber darauf, dass du den resultierenden Sound vom Jamulus Server über deinen Computer abhörst!) Die große Vielfalt an möglichen Hardware-, Software- und Instrumentenkombinationen bedeutet, dass das Einrichten deiner Soundkarte für deine spezielle Konfiguration ansonsten sehr schnell kompliziert werden kann."
```

Comment:
not zu haben, but ein Mischpult zu nutzen

Reply:
Corrected